Libs/Mesh/Mesh.cpp

```diff
@@ -921,7 +991,7 @@ bool Mesh::compareField(const Mesh& other_mesh, const std::string& name1, const
     {
       auto v1(field1->GetTuple(i)[c]);
       auto v2(field2->GetTuple(i)[c]);
-      if (!equalNSigDigits(v1, v2, 5)) {
+      if (!equalNSigDigits(v1, v2, 5, eps)) {
```

Comment:
Significant digits is overall a more accurate test since eps pertains to data range, but as we're noticed with curvature, sometimes the only thing that differs is significant, but it's "close enough." So sometimes it makes sense to use sig digits (most of the time imo) and others to use a sensible epsilon, which varies based on the data being compared. Anyway, the caller of comparisons should decide which to use, not any sort of behind-the-scenes-use-sig-digits-or-epsilon magic.